fsharp-backend/src/BackendOnlyStdLib/LibDarkInternal.fs

```diff
@@ -66,16 +70,18 @@ let modifySchedule (fn : CanvasID -> string -> Task<unit>) =
 
 
 let fns : List<BuiltInFn> =
-  [ { name = fn "DarkInternal" "checkAccess" 0
+  [
+    // WHATISTHIS
+    { name = fn "DarkInternal" "checkAccess" 0
       parameters = []
       returnType = TNull
       description = "TODO"
       fn = internalFn (fun _ -> Ply DNull)
-      sqlSpec = NotYetImplementedTODO
+      sqlSpec = NotQueryable
       previewable = Impure
       deprecated = NotDeprecated }
 
-
+    // fetch email addresses of Dark users who aren't Dark employees
```

Comment:
That's in the description, so unnecessary as a comment. If we should improve the description, I've been making `// CLEANUP` comments to indicate that - we need to keep the descriptions identical for now for tests to pass, but we can update them once we don't care about that anymore.